src/util/Compiler.h

```diff
@@ -42,9 +42,11 @@
 #endif
 
 #if defined(__GNUC__) || defined(__clang__)
-#define gcc_unreachable() __builtin_unreachable()
+#define unreachable() __builtin_unreachable()
+#elif defined(_MSVC_VER)
+#define unreachable() __assume(false)
 #else
-#define gcc_unreachable()
+#define unreachable()
```

Comment:
Dropping this prefix will break the build with a C++23 compiler.

Reply:
Oh, sorry about that. Will fix.